src/Components/FirmwareSelector/index.jsx

```diff
@@ -19,13 +19,12 @@ const blheliModes = blheliSource.getEeprom().MODES;
 
 function FirmwareSelector({
   onCancel,
```

Comment:
Could you please sort the parameters alphabetically. I started doing this in places where I refactored stuff. Just to be consistent.